test/fixtures/integration/llhls-delta-byte-range.m3u8

```diff
@@ -0,0 +1,30 @@
+#EXTM3U
+#EXT-X-TARGETDURATION:10
+#EXT-X-VERSION:3
+#EXT-X-MEDIA-SEQUENCE:0
+#EXT-X-PLAYLIST-TYPE:VOD
+#EXTINF:10,
+#EXT-X-SKIP:SKIPPED-SEGMENTS=3
+#EXTINF:10,
+#EXT-X-BYTERANGE:468684@7108092
+hls_450k_video.ts
+#EXTINF:10,
+#EXT-X-BYTERANGE:444996@7576776
+hls_450k_video.ts
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=45553
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=28823@7622329
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=22444
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=22444
+#EXTINF:10,
+#EXT-X-BYTERANGE:331444@8021772
+hls_450k_video.ts
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=45553@8021772
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=28823
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=22444
+#EXTINF:1.4167,
+#EXT-X-BYTERANGE:44556@8353216
+hls_450k_video.ts
+#EXT-X-PART:DURATION=0.33334,URI="hls_450k_video.ts",BYTERANGE=22444
+#EXT-X-PRELOAD-HINT:TYPE=PART,URI="filePart273.3.mp4",BYTERANGE-LENGTH=2000
+#EXT-X-PRELOAD-HINT:TYPE=PART,URI="filePart273.4.mp4",BYTERANGE-LENGTH=5000,BYTERANGE-START=8377660
```

Comment:
I guess this is causing the out of order byte ranges (and the start used in the other playlist)